### PR TITLE
Added the min_checkpoint_frequency option to the create changefeed docs

### PR DESCRIPTION
**Pull request by kathancox**
Fixes [DOC-5443](https://cockroachlabs.atlassian.net/browse/DOC-5443) & [DOC-5445](https://cockroachlabs.atlassian.net/browse/DOC-5445), [DOC-5682](https://cockroachlabs.atlassian.net/browse/DOC-5682)

Added the `min_checkpoint_frequency` option to the `CREATE CHANGEFEED` docs for v22.1 & v22.2.

### CODE REVIEW

**Comment by kathancox** [comment]
@miretskiy Would you mind reviewing this? And helping me refine the technical detail here?

**Review by miretskiy — APPROVED** [review]

**Comment by kathancox** [comment]
@andf-crl I took you off review for this, I'm just going to look at a few more additions after a meeting discussion. Will add you again when it's ready! Thanks

**Comment by kathancox** [comment]
Hey @miretskiy, I added a slight clarification on the resume and duplicates, let me know if this is sufficient/correct? Thanks!

**Review by miretskiy — APPROVED** [review]

File: v22.1/create-changefeed.md [hunk 1]
```diff
@@ -149,6 +149,7 @@ Option | Value | Description
...
+`min_checkpoint_frequency` | [`INTERVAL`](interval.html) | Controls how often nodes flush their progress to the [coordinating changefeed node](change-data-capture-overview.html#how-does-an-enterprise-changefeed-work). Changefeeds will wait for at least the specified duration before a flush to the sink. This can help you control the flush frequency of higher latency sinks to achieve better throughput. If this is set to `0s`, a node will flush as long as the high-water mark has increased for the ranges that particular node is processing. **Note:** If a changefeed is resumed, then `min_checkpoint_frequency` is the amount of time that changefeed will need to catch up. That is, it could emit duplicate messages during this time.<br><br>**Default:** is `30s`.
```
Comment (andf-crl) [inline v22.1/create-changefeed.md:152]: Nit: recommend the following to match other defaults on this page:

```
**Default:** `30s` 
```

(omitting `is`, and `.`)

**Review by andf-crl — APPROVED** [review]
LGTM with one tiny nit. Thanks!

File: v22.1/create-changefeed.md (hunk 1 above)
Comment (kathancox) [reply v22.1/create-changefeed.md:152]: oh whoops! Definitely a typo, thanks for catching that!

**Pull request merged**